doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,599 @@
+      See :ref:`Zephyr documentation about using zephyrrc files <zephyr:env_vars_zephyrrc>` for more information.
+
+You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.
+The west command programs the development kit by using nrfjprog by default.
+For more information on nrfjprog, see `Programming SoCs with nrfjprog`_.
+
+.. _additional_deps:
+
+System-wide installation
+************************
+
+System-wide installation is an alternative to the recommended installation methods using the |nRFVSC| or nRF Util.
+It gives you more control over each of the required tools, but requires more familiarity with Zephyr and with each of the tools.
+
+To install the |NCS| system-wide, complete the following steps:
+
+1. Follow the steps 1 and 2 in Zephyr's :ref:`zephyr:getting_started` to update your operating system and install dependencies.
+#. Install west.
+   Expand the section below to see the commands.
+
+   .. toggle::
+
+      .. note::
+
+         Use a command prompt window with administrator rights to ensure that Python is initialized.
+
+      .. tabs::
+
+         .. group-tab:: Windows
+
+            Enter the following command in a command-line window:
+
+            .. code-block:: console
+
+               pip3 install west
+
+            .. note::
+               Ensure the west location is added to the path in environmental variables.
+
+         .. group-tab:: Linux
+
+            Enter the following command in a terminal window:
+
+            .. code-block:: console
+
+               pip3 install --user west
+               echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+               source ~/.bashrc
+
+         .. group-tab:: macOS
+
+            Enter the following command in a terminal window:
+
+            .. code-block:: console
+
+               pip3 install west
+
+      ..
+
+   ..
+
+#. Get the |NCS| code as described in :ref:`cloning_the_repositories`.
+#. Install the Python dependencies.
+   Expand the section below to see the commands.
+
+   .. toggle::
+
+      .. note::
+
+         You might run into Python package incompatibilities when installing dependencies at a system or user level.
+         For this reason, it is suggested to use `Python virtual environments`_.
+
+      .. tabs::
+
+         .. group-tab:: Windows
+
+            1. Use the following command to install the Python ``venv`` package:
+
+               .. code-block:: bash
+
+                  pip install virtualenv
+
+            #. Create a new virtual environment:
+
+               .. code-block:: bash
+
+                  cd %HOMEPATH%
+                  python -m venv ncs\.venv
+
+            #. Activate the virtual environment:
+
+               .. code-block:: bash
+
+                  :: cmd.exe
+                  ncs\.venv\Scripts\activate.bat
+                  :: PowerShell
+                  ncs\.venv\Scripts\Activate.ps1
```

Comment:
Let's get rid of powershell unless someone can confirm it is supported and works (I have heard of no such support)